### PR TITLE
Sync the memory numbers with the numbers in FRO.

### PR DESCRIPTION
**Pull request by leegeth**
Sync the memory numbers with the numbers in Freertos.org at https://www.freertos.org/http/index.html

### CODE REVIEW

**Comment by leegeth** [comment]
image

**Review by archigup — DISMISSED** [review]

File: docs/doxygen/pages.dox [hunk 1]
```diff
@@ -19,31 +19,27 @@ Feature of HTTP/1.1 not supported in this library:
...
     <tr>
         <td>http_parser.c</td>
-        <td>38.9K</td>
-        <td>22.2K</td>
-        <td>17.6K</td>
+        <td><center>15.7K</center></td>
+        <td><center>13.0K</center></td>
     </tr>
     <tr>
         <td><b>Total estimates</b></td>
```
Comment (leegeth) [inline docs/doxygen/pages.dox:40]:
```suggestion
        <td><b>Total estimate</b></td>
```

File: docs/doxygen/pages.dox [hunk 2]
```diff
@@ -19,31 +19,27 @@ Feature of HTTP/1.1 not supported in this library:
...
     <tr>
         <td>core_http_client.c</td>
-        <td>16.4K</td>
-        <td>4.7K</td>
-        <td>4.0K</td>
+        <td><center>3.1K</center></td>
+        <td><center>2.5K</center></td>
     </tr>
     <tr>
         <td>http_parser.c</td>
```
Comment (leegeth) [inline docs/doxygen/pages.dox:35]:
```suggestion
        <td>http_parser.c (third-party utility)</td>
```

File: docs/doxygen/pages.dox [hunk 3]
```diff
@@ -19,31 +19,27 @@ Feature of HTTP/1.1 not supported in this library:
 
 <table>
     <tr>
-        <td colspan="4"><center><b>Code size of HTTP Client library files (sizes generated with [GCC for ARM Cortex-M toolchain](https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads/9-2020-q2-update))</b></center></td>
+        <td colspan="3"><center><b>Code size of HTTP Client library(example generated with GCC for ARM Cortex-M)</b></center></td>
```
Comment (archigup) [inline docs/doxygen/pages.dox:22]:
```suggestion
        <td colspan="3"><center><b>Code size of HTTP Client library (example generated with GCC for ARM Cortex-M)</b></center></td>
```

**Review by archigup — APPROVED** [review]

**Review by cobusve — APPROVED** [review]

**Pull request merged**